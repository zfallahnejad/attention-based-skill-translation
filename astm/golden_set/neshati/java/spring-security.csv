241990,spring-security
103154,spring-security
618087,spring-security
280244,spring-security
1259109,spring-security
1677948,spring-security
1276804,spring-security
139985,spring-security
2696260,spring-security
1463522,spring-security
708434,spring-security
3510410,spring-security
4249,spring-security
158658,spring-security
