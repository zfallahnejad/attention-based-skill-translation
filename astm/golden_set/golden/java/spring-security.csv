241990,73,45,0.6186440677966102
103154,41,18,0.6949152542372882
618087,35,19,0.6481481481481481
280244,31,25,0.5535714285714286
1259109,28,22,0.56
1677948,26,30,0.4642857142857143
1276804,20,7,0.7407407407407407
139985,18,20,0.47368421052631576
2696260,17,10,0.6296296296296297
1463522,17,25,0.40476190476190477
708434,16,1,0.9411764705882353
3510410,16,12,0.5714285714285714
4249,13,11,0.5416666666666666
158658,12,7,0.631578947368421
2898867,10,8,0.5555555555555556
